### PR TITLE
 MDCT-2420: Disable All Fields + Buttons if MLR Report is Locked

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

If an MLR report is in a `Locked` state, all of its action buttons and fields should be disabled. 

### Related ticket(s)

MDCT-2420

---
### How to test

- Log in as a state user
- Create a new MLR report
- Submit the report

Verify that all fields and buttons are disabled ✨ 

- Log in as an admin user
- Navigate to the submitted report

Ditto, everything is now disabled.

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/ui-src/src/components/tables/EntityRow.tsx [hunk 1]
```diff
@@ -141,7 +149,7 @@ const sx = {
   deleteButton: {
     padding: "0",
     background: "white",
-    "&:hover": {
+    "&:hover:disabled": {
       background: "white",
     },
```
Comment (braxex) [inline services/ui-src/src/components/tables/EntityRow.tsx:154]: Question: Do we want this behavior _only_ when hovering _and_ disabled, or both when hovering _and_ when hovering + disabled?

Comment (braxex) [reply services/ui-src/src/components/tables/EntityRow.tsx:154]: _applies to other locations in PR_

Comment (karla-vm) [reply services/ui-src/src/components/tables/EntityRow.tsx:154]: Both!

**Review by keeysnc — APPROVED** [review]
Looks good to me! The only thing I noticed was code climate flagged a lower percentage on diff coverage. It seems like it might be an out of scope issue though?

**Comment by karla-vm** [comment]
> Looks good to me! The only thing I noticed was code climate flagged a lower percentage on diff coverage. It seems like it might be an out of scope issue though?

Yep! I think there's an issue with CodeClimate for this particular branch, so we're going to have a separate effort to look into it :)

**Pull request merged**